Source/santad/DataLayer/SNTRuleTable.m

```diff
@@ -133,7 +133,8 @@ - (SNTRule *)ruleForBinarySHA256:(NSString *)binarySHA256
 
 #pragma mark Adding
 
-- (BOOL)addRules:(NSArray *)rules cleanSlate:(BOOL)cleanSlate error:(NSError **)error {
+- (BOOL)addRules:(NSArray *)rules cleanSlate:(BOOL)cleanSlate
+           error:(NSError * __autoreleasing *)error {
```

Comment:
I would double check this doesn't release too early.

Reply:
It doesn't, this is the fix recommended by clang (see [here](http://lists.llvm.org/pipermail/cfe-commits/Week-of-Mon-20170213/185409.html))